.eslintrc.js

```diff
@@ -1,10 +1,110 @@
 module.exports = {
+	root: true,
+	globals: {
+		__DEV__: true,
+		process: true,
+		window: true,
+		document: true
+	},
 	env: {
 		node: true
 	},
-	extends: ['enact', 'plugin:prettier/recommended', 'prettier'],
-	plugins: ['import'],
+	extends: ['eslint:recommended', 'plugin:prettier/recommended', 'prettier'],
+	parser: '@babel/eslint-parser',
+	parserOptions: {
+		ecmaFeatures: {
+			jsx: true
+		},
+		ecmaVersion: 'latest',
+		sourceType: 'module',
+		requireConfigFile: false
+	},
+	plugins: ['@babel', 'import'],
 	rules: {
+		'block-scoped-var': 'warn',
+		curly: ['warn', 'multi-line'],
+		eqeqeq: ['warn', 'smart'],
+		'new-parens': 'warn',
+		'no-alert': 'warn',
+		'no-array-constructor': 'warn',
+		'no-caller': 'error',
+		'no-catch-shadow': 'error',
+		'no-cond-assign': ['warn', 'except-parens'],
+		'no-constant-condition': 'warn',
+		'no-control-regex': 'off',
+		'no-debugger': 'off',
+		'no-div-regex': 'warn',
+		'no-empty': 'warn',
+		'no-eval': 'warn',
+		'no-extend-native': 'warn',
+		'no-extra-bind': 'warn',
+		'no-extra-boolean-cast': 'warn',
+		'no-extra-semi': 'off',
+		'no-fallthrough': 'warn',
+		'no-floating-decimal': 'warn',
+		'no-func-assign': 'warn',
+		'no-implied-eval': 'warn',
+		'no-inner-declarations': 'off',
+		'no-irregular-whitespace': 'warn',
+		'no-iterator': 'error',
+		'no-label-var': 'error',
+		'no-labels': 'error',
+		'no-mixed-spaces-and-tabs': ['warn', 'smart-tabs'],
+		'no-global-assign': 'error',
+		'no-unsafe-negation': 'error',
+		'no-new-func': 'error',
+		'no-new-object': 'warn',
+		'no-new-wrappers': 'warn',
+		'no-octal-escape': 'warn',
+		'no-proto': 'error',
+		'no-redeclare': [
+			'error',
+			{
+				builtinGlobals: true
+			}
+		],
```

Comment:
Why is the indentation different from enact config like below?

```suggestion
		'no-redeclare': ['error', {
			builtinGlobals: true
		}],
```

Reply:
I see 3 more.

Reply:
Those 2 lines occurred lint errors

/home/taeyoung.hong/webos_build_server/WRO-697/babel-preset-enact/.eslintrc.js
  60:1   error  Insert `↹`                                   prettier/prettier
  61:3   error  Replace `}` with `↹}⏎↹↹`                     prettier/prettier

I will further analyze why those new errors occurred.

Reply:
These changes are due to a lint error detected in the prettier/prettier.
The original in eslint-config-enact is not detected because there is no prettier, but it is detected in the babel-preset-enact.
There are three ways to solve this problem.

1. Exclude .eslintrc.js from lint
2. Remove the prettier
3. Apply changes

Reply:
Let's keep this as-is.